metrics/metrics.go

```diff
@@ -58,6 +61,12 @@ var ConsumingErrors = promauto.NewCounter(prometheus.CounterOpts{
 	Help: "The total number of errors during consuming messages from Kafka",
 })
 
+// SkippedEmptyReports shows the total number of consumed messages not processed as the rule execution resulted in an empty report
+var SkippedEmptyReports = promauto.NewCounter(prometheus.CounterOpts{
```

Comment:
maybe you would also like to update the grafana dashboard with this metric?

Reply:
that's the plan =)